### ISSUE TITLE
按照教程训练的主体识别效果差

### ISSUE THREAD

**Issue by itues** [PaddlePaddle/PaddleClas#2797]
我需要训练关于鞋子的主体识别，按照这个教程操作的：https://github.com/PaddlePaddle/PaddleClas/blob/release/2.5/docs/zh_CN/training/PP-ShiTu/mainbody_detection.md#41-%E6%8E%A8%E7%90%86%E6%A8%A1%E5%9E%8B%E5%87%86%E5%A4%87
 1. 使用PaddleDetection-release-2.6
 2. 训练环境信息：
  a. 具体操作系统Windows
  b. Python版本号：3.6
  c. CUDA/cuDNN版本：CUDA10.2/cuDNN 7.6.5等
 3.picodet_lcnet_x2_5_640_mainbody.yml配置如下。其他文件中只修改了学习率为base_lr: 0.00025。

_BASE_: [
  '../../../../runtime.yml',
  '../../_base_/picodet_esnet.yml',
  '../../_base_/optimizer_100e.yml',
  '../../_base_/picodet_640_reader.yml',
]

pretrain_weights: premodels/LCNet_x2_5_ssld_pretrained.pdparams
#pretrain_weights: https://paddledet.bj.bcebos.com/models/pretrained/LCNet_x2_5_ssld_pretrained.pdparams
weights: output/picodet_lcnet_x2_5_640_mainbody/model_final
find_unused_parameters: True
use_ema: true
cycle_epoch: 20
snapshot_epoch: 8

PicoDet:
  backbone: LCNet
  neck: CSPPAN
  head: PicoHead

LCNet:
  scale: 2.5
  feature_maps: [3, 4, 5]

metric: COCO
num_classes: 1

TrainDataset:
  !COCODataSet
    image_dir: JPEGImage
    anno_path: ImageSets/voc_train.json
    dataset_dir: dataset/MyDataset/shoes/  
    data_fields: ['image', 'gt_bbox', 'gt_class', 'is_crowd']

EvalDataset:
  !COCODataSet
    image_dir: JPEGImage
    anno_path: ImageSets/voc_val.json
    dataset_dir: dataset/MyDataset/shoes/

TestDataset:
  !ImageFolder
    anno_path: ImageSets/voc_val.json # also support txt (like VOC's label_list.txt)   
    
    

出现的问题：
 1.预测的结果杂乱目标太多了，不知道哪里出了问题。希望指导下。如下图
 2.按照文档修改了voc_train.json和voc_val.json里面的为"categories": [{"supercategory": "foreground", "id": 1, "name": "foreground"}]。只有前景类型，但是预测出来的都是‘person’，这是怎么回事？

![image](https://github.com/PaddlePaddle/PaddleClas/assets/86935392/8ff5b30d-137c-4850-be19-95955ec1c06f)

**Comment by xiaohui0225** [comment]
遇到了同样的问题，请问你解决了吗？
